package.json

```diff
@@ -65,6 +65,7 @@
   },
   "scripts": {
     "build": "./scripts/build",
+    "clean": "rm -rf /tmp/.yarn-bazel-cache/ && bazel clean",
```

Comment:
You shouldn't need this anymore, I fixed the underlying issue

Reply:
ok, I will remove it